### PR TITLE
Tizio/one gui to rule them all

### LINKED ISSUE #66: Final RealityPlayer parameters tuning with use of gui sliders
In the attached branch you can find Scene_slopy_calibration where you have a GUI that let's you control in game using sliders the player's parameters. It is not supposed to become more refined than this keeping in mind that as soon as the parameters are chosen it will be thrown away.

Mind that this branch works on a copy of allplayer, that is AllPlayerCalibration, and the RealityPlayer of this prefab uses the script RealityMovementCalibration not to break the old one using the gui. I suggest that once the parameter tuning together with the feature fixing is finished we either copy the new fixes to the old one or rename this one and remove/disable the calibration gui

### PR DESCRIPTION
**Pull request by TizioMaurizio**
WORK IN PROGRESS

### CODE REVIEW

**Comment by TizioMaurizio** [comment]
Maybe this won't be merged but I just wanted to try out PR drafts since I just learnt they exist from another project on GitLab.
I suggest that we open PR drafts from issues with active branch not to forget that the issue is already work in progress.

**Comment by azazellus** [comment]
Important! _CalibrationMenu();_ must be added in the Update of RealityMovementCalibration script, otherwise the sliders don't work :(

**Comment by TizioMaurizio** [comment]
RealityPlayerCalibration is a bit dirty but it needs to be reworked anyway, the utils folders are the far west but what is in utils stays in utils

File: Assets/Code/Scripts/NoclipRealityManagement/RealityMovementCalibration.cs [hunk 1]
```diff
@@ -0,0 +1,489 @@
...
+            _OnSlope = OnSlope(); // used to debug // TODO remove
+            Grounded = _grounded; // used to debug // TODO remove
+        
+            UserInput();
+            StateHandler();
+            MovePlayer();
+            SpeedControl();
+
+            // handle drag
+            if (_grounded && !_commitJump)
```
Comment (stefanofossati) [inline Assets/Code/Scripts/NoclipRealityManagement/RealityMovementCalibration.cs]: Yesterday we tried to add this part in order to have the drag active only when any keys aren't pressed.
@AleDeNic said that is better in this way.

```suggestion
            if (_grounded && && _horizontalInput == 0 && _verticalInput == 0 && !_commitJump)
```

**Review by stefanofossati — COMMENTED** [review]
A part of the mouse movement in reality mode, all works in 0_main scene

**Comment by TizioMaurizio** [comment]
> A part of the mouse movement in reality mode, all works in 0_main scene

was the mouse sensistivity slider from the gui different from zero?

**Review by stefanofossati — DISMISSED** [review]
Ok the mouse works :)

**Review by stefanofossati — APPROVED** [review]

**Pull request merged**